### ISSUE TITLE
Git link is incorrect.

### ISSUE THREAD

**Issue by DailyDreaming** [HumanCellAtlas/matrix-service#319]
Using this for the first time at the face-to-face and following instructions:

```
quokka@qcore:~/git$ git clone git@github.com:HumanCellAtlas/matrix-service.git
```

```
Cloning into 'matrix-service'...
The authenticity of host 'github.com (140.82.118.3)' can't be established.
RSA key fingerprint is SHA256:nThbg6kXUpJWGl7E1IGOCspRomTxdCARLviKw6E5SY8.
Are you sure you want to continue connecting (yes/no)? yes
Warning: Permanently added 'github.com,140.82.118.3' (RSA) to the list of known hosts.
git@github.com: Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

Command should be `git clone https://github.com/HumanCellAtlas/matrix-service.git` for normal users.

**Comment by brianraymor** [comment]
Specifically, the incorrect instructions are in [README.md](https://github.com/HumanCellAtlas/matrix-service#developer-environment-setup).

**Comment by brianraymor** [comment]
Closed by #320
